examples/bls.rs

```diff
@@ -0,0 +1,8 @@
+use ethereum_consensus::crypto;
```

Comment:
can likely drop this import if we only use `SecretKey` here

Reply:
fixed